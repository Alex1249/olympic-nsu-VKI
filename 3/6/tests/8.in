16 2 10
a1c.01d