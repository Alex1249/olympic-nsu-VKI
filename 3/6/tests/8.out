101000011100.0000000111
